config/calibrate.py

```diff
@@ -21,10 +21,6 @@
 config.photoCal.match.referenceSelection.magLimit.maximum = 22.0
 config.photoCal.colorterms.load(os.path.join(obsConfigDir, 'colorterms.py'))
 
-# Number of bright stars to use. Sets the max number of patterns that can be tested.
-# This config matches obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
-config.astrometry.matcher.numBrightStars = 150
-
```

Comment:
It looks like this is being removed to keep this config in-line with obs_subaru, where I note that you're also removing this config override. This means that both DECam and HSC will default to using the task default, which is 200. Can you confirm this is the case?

Reply:
The task default is being changed to 150:

https://github.com/lsst/meas_astrom/pull/185/files